infra/docker-compose/dev/turing/db-migrations

```diff
@@ -0,0 +1 @@
+../../../../api/db-migrations/
```

Comment:
I vaguely remember that there was an issue with using symlinks with docker. Have you verified that this works as expected?

Reply:
What was the issue? I just did `docker-compose up -d` and it works.

Reply:
Okay 👌